.github/workflows/os-compability-test.yaml

```diff
@@ -0,0 +1,84 @@
+name: os-compability-test
+run-name: OS compability test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: make build/*
+    runs-on: ubuntu-latest
+    # timeout-minutes: 10 TODO: Uncomment on production runner
+    container: 
+      image: public.ecr.aws/gravitational/teleport-buildbox-centos7:teleport11
+      env:
+        GOCACHE: /tmp/gocache
+      volumes:
+        - ${{ github.workspace }}:/workspace      
+    steps:
+      - name: Install git 1.18+
```

Comment:
This workflow does not need to login to Google Cloud, so, I decided that it would hurt less to expand everything here then to call login-to-cloud every time. Let's wait for a decision with logs uploads before I change it here.